core/utils/parsing.ts

```diff
@@ -21,8 +21,8 @@ import * as colourUtils from './colour.js';
  *
  * @param message Text which might contain string table references and
  *     interpolation tokens.
- * @param parseInterpolationTokens Option to parse numeric
- *     interpolation tokens (%1, %2, ...) when true.
+ * @param parseInterpolationTokens Option to parse numeric interpolation
+ *     tokens (%1, %2, ...) and newline characters when true.
  * @returns Array of strings and numbers.
  */
 function tokenizeInterpolationInternal(
```

Comment:
Could you add tests for this new functionality to `tests/mocha/utils_test.js`? (sorry for the terrible file naming lol)